contracts/seed/Seed.sol

```diff
@@ -44,6 +44,7 @@ contract Seed {
     bytes public metadata; // IPFS Hash
 
     uint256 internal constant PRECISION = 10**18; // used for precision e.g. 1 ETH = 10**18 wei; toWei("1") = 10**18
+    uint8 private _seedTokenDecimal; // for adjustment of difference of power of 10 between project and funding token
```

Comment:
_seedTokenDecimals (plural) (should be plural where ever we refer to a count of decimals)

can we say "// for adjustment of difference of power of 10 between project and funding token, typically 18"